src/sentry/api/serializers/models/group.py

```diff
@@ -44,7 +43,7 @@
 from sentry.tsdb.snuba import SnubaTSDB
 from sentry.utils.db import attach_foreignkey
 from sentry.utils.safe import safe_execute
-from sentry.utils.compat import zip
+from sentry.utils.compat import map, zip
```

Comment:
Surprised that this shadowing of built-ins not triggering any lint issues.

Reply:
It's desired: https://github.com/getsentry/sentry-flake8/pull/8